compiler/test/unit/lexer/location_offset.d

```diff
@@ -557,7 +557,7 @@ static foreach (tok; __traits(allMembers, TOK))
         @(tests[tok].description)
         unittest
         {
-            const newCode = "first_token " ~ tests[tok].code;
+            const newCode = "first_token " ~ tests[tok].code ~ '\0';
```

Comment:
I don't know why this PR exposed this missing '\0', but okay.

Reply:
This is a `string`, but a few lines below `newCode.ptr` is used. Without the `\0` at the end, it produced a seg fault.